pkg/config/defaults.go

```diff
@@ -159,6 +159,19 @@ const (
 	LakefsEmailBaseURLKey      = "email.lakefs_base_url"
 
 	UIEnabledKey = "ui.enabled"
+
+	GravelerRepositoryCacheSizeKey       = "graveler.repository_cache.size"
+	DefaultGravelerRepositoryCacheSize   = 1000
+	GravelerRepositoryCacheExpiryKey     = "graveler.repository_cache.expiry"
+	DefaultGravelerRepositoryCacheExpiry = 5 * time.Second
+	GravelerRepositoryCacheJitterKey     = "graveler.repository_cache.jitter"
+	DefaultGravelerRepositoryCacheJitter = DefaultGravelerRepositoryCacheExpiry / 2
+	GravelerCommitCacheSizeKey           = "graveler.commit_cache.size"
+	DefaultGravelerCommitCacheSize       = 50000
+	GravelerCommitCacheExpiryKey         = "graveler.commit_cache.expiry"
+	DefaultGravelerCommitCacheExpiry     = 30 * time.Second
+	GravelerCommitCacheJitterKey         = "graveler.commit_cache.jitter"
+	DefaultGravelerCommitCacheJitter     = DefaultGravelerCommitCacheExpiry / 2
```

Comment:
This seems very high, we want a long expiry time.  And if it's inconsistent, FindMergeBase may take very different times to do the exact same thing.
```suggestion
	DefaultGravelerCommitCacheJitter     = 2 * time.Second
```

Reply:
Agree - I also not sure jitter is needed for these type of cache